pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/Statistics.sql.stg

```diff
@@ -0,0 +1,87 @@
+group Statistics;
+
+getParticipantStatistics() ::= <<
+select
+    case when ug.user_governance_id is null then 'self' else 'governed' end as user_type,
+    count(1) as user_count
+from
+    user_study_enrollment se
+    left join umbrella_study us on se.study_id = us.umbrella_study_id
+    left join user_governance ug on ug.participant_user_id = se.user_id
+    left join user_study_governance usg on usg.umbrella_study_id = us.umbrella_study_id
+        and usg.user_governance_id = ug.user_governance_id and ug.is_active = 1
+where
+    us.umbrella_id = :studyId and
+    se.enrollment_status_type_id = 2
+group by 1
+>>
+
+getAnswersDistributionByQueryStableId() ::= <<
+select 
+    po.picklist_option_stable_id as answer, 
+    count(1) as count 
+from 
+question_stable_code qsc 
+join umbrella_study us on qsc.umbrella_study_id = us.umbrella_study_id 
+    join question q on qsc.question_stable_code_id = q.question_stable_code_id 
+    join picklist_question pq on pq.question_id = q.question_id 
+    join answer a on a.question_id = q.question_id 
+    join activity_instance ai on ai.activity_instance_id = a.activity_instance_id 
+    join user_study_enrollment se on ai.participant_id = se.user_id 
+    join picklist_option__answer poa on poa.answer_id = a.answer_id 
+    join picklist_option po on po.picklist_option_id = poa.picklist_option_id 
+    join revision as r on r.revision_id = po.revision_id 
+where 
+    us.umbrella_id = :studyId and 
+    qsc.stable_id = :stableId and 
+    se.enrollment_status_type_id = 2 and
+    r.end_date is null 
```

Comment:
Instead of just getting the latest revision of the `picklist_option`, I think we should get the revision that matches the `activity_instance`. Do this with the following:
```
and r.start_date \<= ai.created_at
and (r.end_date is null or ai.created_at \< r.end_date)
```
What this does is, it looks for a particular revision that is "active" at the time the activity instance is created. Note the `\` (backslash) characters, those are needed to escape the `<` in `.stg` files since `<` is part of the syntax.

Reply:
@YuriyPavlovBI I still think adding those `where` clauses to restrict the time range will be safer.

Reply:
Sorry, missed this comment. It was hidden for some reason. Time range check added